### DISCUSSION TITLE
FLUX.1-dev-Controlnet-Inpainting-Alpha

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/stable-diffusion-webui-forge#1809] [General]
https://huggingface.co/alimama-creative/FLUX.1-dev-Controlnet-Inpainting-Alpha/tree/main

**Comment by Gushousekai195** [comment]
Another vital component of my workflow. I just wish we weren’t waiting so long to get Flux Controlnet for Forge. -_-

**Comment by evanheckert** [comment]
Oh great, would love to see this usable in Forge!

**Comment by nitinmukesh** [comment]
Can't wait for Flux controlnet to work in Forge.
